assets/js/controllers/typewriter_controller.js

```diff
@@ -0,0 +1,18 @@
+import { Controller } from '@hotwired/stimulus';
+import Typewriter from '../feature/Typewriter';
+
+export default class extends Controller {
+    connect() {
+        const text = [
```

Comment:
💡 **suggestion:** Tan qu'à utiliser Stimulus, il vaudrait mieux en profiter pour passer ce genre de valeur directement dans le HTML pour rendre ce controller plus réutilisable.

Par exemple, afin de configurer le texte à prendre en compte, voici ce que tu peux faire : 

```diff
diff --git a/assets/js/controllers/typewriter_controller.js b/assets/js/controllers/typewriter_controller.js
index 542f9d4619..d5ffc7abe4 100644
--- a/assets/js/controllers/typewriter_controller.js
+++ b/assets/js/controllers/typewriter_controller.js
@@ -2,17 +2,15 @@ import { Controller } from '@hotwired/stimulus';
 import Typewriter from '../feature/Typewriter';
 
 export default class extends Controller {
+    static values = {
+        text: Array,
+    }
+
     connect() {
-        const text = [
-            '"Salut, je suis l\'assistant qui vous aide à concevoir le brief de votre projet. ',
-            'À l\'issu de notre échange, je vous enverrai par mail un brief finalisé prêt à l\'emploi !',
-            'Êtes-vous prêts à commencer ?',
-            'Première étape, je vous invite à choisir le contexte de votre besoin."'
-        ];
         const speed = 100;
         const scrollAt = 20;
         const destinationId = 'typedtext';
-        const typewriter = new Typewriter(text, speed, scrollAt, destinationId);
+        const typewriter = new Typewriter(this.textValue, speed, scrollAt, destinationId);
         typewriter.startTyping();
     }
 }
diff --git a/templates/site/home.html.twig b/templates/site/home.html.twig
index ed6c70c8c0..b5069b5af9 100644
--- a/templates/site/home.html.twig
+++ b/templates/site/home.html.twig
@@ -71,7 +71,14 @@
                         <p id="typedtext"></p>
                     </div>
 
-                    <ul class="ia-home__needs" {{ stimulus_controller('typewriter') }}>
+                    <ul class="ia-home__needs" {{ stimulus_controller('typewriter', {
+                        text: [
+                            '"Salut, je suis l\'assistant qui vous aide à concevoir le brief de votre projet. ',
+                            'À l\'issu de notre échange, je vous enverrai par mail un brief finalisé prêt à l\'emploi !',
+                            'Êtes-vous prêts à commencer ?',
+                            'Première étape, je vous invite à choisir le contexte de votre besoin."'
+                        ]
+                    }) }}>
                         <li class="ia-home__need">
                             <div>
                                 <h3>J'ai déjà mon idée et j'ai besoin d'une expertise</h3>
```

➜ tu exposes des data attributes sur l'élément HTML où tu souhaites câbler le controller : 
![image](https://github.com/Elao/elao_/assets/2211145/dd945368-a62e-478e-af17-6fc8e2d08c26)

et tu les décrits dans la propriété `values` de ton controller.
➜ tu peux alors accéder à leur valeur directement avec `this.[nom-de-ton-attribue]Value`

Cf https://stimulus.hotwired.dev/reference/values
![image](https://github.com/Elao/elao_/assets/2211145/7a257704-750e-4e9e-b3b2-99d0822a4cfb)